src/components/ui/Swap/SwapSettingModal.tsx

```diff
@@ -72,10 +72,7 @@ export const SwapSettingModal = (props: SwapSettingProps) => {
           <NumberInput
             classNames={{
               root: "w-auto",
-              input: clsx(
-                dogica.className,
-                "text-center  md:text-start bg_turq text-black text-xs h-[42px] border-transparent rounded-none"
-              ),
+              input: clsx(dogica.className, classes.inputField),
```

Comment:
I've added a util function called `cn()`. [You can find it in here.](https://github.com/weweswap/app-page/blob/main/src/utils/index.ts#L5)

It uses both `clsx` and `tailwind-merge` package. So, we won't have conflict in the tailwind class names. Always apply the last class.